dazel.py

```diff
@@ -121,6 +123,7 @@ def from_config(cls):
                 dockerfile=config.get("DAZEL_DOCKERFILE", DEFAULT_LOCAL_DOCKERFILE),
                 repository=config.get("DAZEL_REPOSITORY", DEFAULT_REMOTE_REPOSITORY),
                 directory=config.get("DAZEL_DIRECTORY", DEFAULT_DIRECTORY),
+                build_directory=config.get("DAZEL_BUILD_DIRECTORY", DEFAULT_BUILD_DIRECTORY),
```

Comment:
To make this backwards compatible, can you please make build_directory fallback to directory rather than than to the default?